docs/external-cluster.md

```diff
@@ -0,0 +1,82 @@
+# Use external Virtink infrastructure cluster
```

Comment:
For titles and headers, cap them like "Use External Virtink Infrastructure Cluster"